### PR TITLE
Feature/display sent messages immediately

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

**Review by Dinek007 — APPROVED** [review]

**Review by EmiM — APPROVED** [review]

File: packages/nectar/src/sagas/messages/updateMessagesSendingStatus/updateMessagesSendingStatus.saga.test.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      'Message',
+      { identity: alice }
+    )).payload.message
+  })
+
+  test('remove pending message status', async () => {
+    const reducer = combineReducers(reducers)
+    await expectSaga(
+      updateMessagesSendingStatusSaga,
+      publicChannelsActions.incomingMessages({
```
Comment (holmesworcester) [inline packages/nectar/src/sagas/messages/updateMessagesSendingStatus/updateMessagesSendingStatus.saga.test.ts:50]: Is it confusing that we're adding these messages to a pile called incoming messages? Or is that not what's happening here?

**Review by holmesworcester — APPROVED** [review]

File: packages/nectar/src/sagas/messages/updateMessagesSendingStatus/updateMessagesSendingStatus.saga.test.ts (hunk 1 above)
Comment (siepra) [reply packages/nectar/src/sagas/messages/updateMessagesSendingStatus/updateMessagesSendingStatus.saga.test.ts:50]: It's only calling the action that updates redux store. It's important to do it this way here as this action triggers the exact saga we want to test. In real life this action is being called by the socket event that comes from waggle (but in this kind of tests we don't have waggle running)

Comment (siepra) [reply packages/nectar/src/sagas/messages/updateMessagesSendingStatus/updateMessagesSendingStatus.saga.test.ts:50]: It's just an argument provided to the saga we're testing

Comment (siepra) [reply packages/nectar/src/sagas/messages/updateMessagesSendingStatus/updateMessagesSendingStatus.saga.test.ts:50]: This argument comes in the form of redux's action creator

**Pull request merged**